### ISSUE TITLE
Unexpected token q

### ISSUE THREAD

**Issue by rohan5894** [devbridge/jQuery-Autocomplete#184]
I getting the Unexpected token q error while using the Ajax Autocomplete.

Here is my code like:
$('#autocomplete-ajax').autocomplete({
        serviceUrl:'ajax/ajax.MainSearch.php',
        onSelect: function(suggestion) {
            $('#selction-ajax').html('You selected: ' + suggestion.value + ', ' + suggestion.data);
        },
        onHint: function (hint) {
            $('#autocomplete-ajax-x').val(hint);
        },
        onInvalidateSelection: function() {
            $('#selction-ajax').html('You selected: none');
        }
    });

Can anyone please help?

Thanks

**Comment by amber19** [comment]
Hi Tom, is there a help forum to resolve errors? I'm trying to implement your autocomplete in a Coldfusion project, reading from a CFC or even a cfm file that returns a json object in this format:
" {query:'chi',suggestions:['value1','value2'],data:[11111, 22222]}" and I get the same "unexpected token q error"  

When I eliminated the 'q' element from json, and my response looked like this:
"{suggestions:['value1','value2'],data:[11111, 22222]}"  it threw: "Unexpected token s"

When I eliminated 'suggestions' text from the string, and my response looked like this:"{['value1','value2'],data:[11111, 22222]}"  it threw: "Unexpected token  [ "

What does it really expect right after the opening bracket '{'?

This is how my call back looks like: 
onSelect: function(suggestion){           
            $('#query').val(suggestion.value);
            $('#answer').val(suggestion.data);
}
query is the field linked with autocomplete, answer is another input field I try to place data value in.

Any suggestion/help will be greatly appreciated, thanks much!
